autoscale-scaler-dockerswarm/pom.xml

```diff
@@ -72,7 +61,17 @@
             <artifactId>mockito-core</artifactId>
             <scope>test</scope>
         </dependency>
-
-       
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.jayway.jsonpath</groupId>
+            <artifactId>json-path</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+        </dependency> 
```

Comment:
- Rouge space
- Keep compile-time dependencies together, above the test-dependencies